src/views/AddItem.jsx

```diff
@@ -1,3 +1,77 @@
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	const [newItem, setNewItem] = useState('');
+	const [nextPurchaseTime, setPurchaseTime] = useState(7);
+	const [statusMessage, setStatusMessage] = useState('');
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		let listId = 'my test list';
+
+		const item = await addItem(listId, {
+			itemName: newItem,
+			daysUntilNextPurchase: nextPurchaseTime,
+		});
+
+		if (item.id) {
+			setStatusMessage('Item Successfully Added!');
+			setNewItem('');
+			setPurchaseTime(7);
+		} else {
+			setStatusMessage('Item Not Added!');
+		}
+	};
+	return (
+		<>
+			<form onSubmit={handleSubmit}>
+				<div>
+					<label htmlFor="name">Item Name</label>
+					<input
+						type="text"
+						id="name"
+						placeholder="Enter Item to Purchase"
+						value={newItem}
+						onChange={(e) => setNewItem(e.target.value)}
+					/>
+				</div>
+
+				<div>
+					<fieldset onChange={(e) => setPurchaseTime(e.target.value)}>
+						<legend>How soon will you buy this again?</legend>
+
+						<div>
+							<input
+								type="radio"
+								id="soon"
+								name="duration"
+								value={7}
+								defaultChecked
```

Comment:
example of a controlled radio button: 
```
onChange={(e) => setPurchaseTime(e.target.value)}
checked={7 === parseInt(nextPurchaseTime)}
```
and just remove the `onChange` function from the `fieldset`.
What we are doing here, is just deciding if a radio button is checked or not based on its value (7 for this input) and our current state.